src/frameName.cpp

```diff
@@ -135,13 +139,20 @@ char* FrameName::truncate(char* name, int max_length) {
 
 const char* FrameName::decodeNativeSymbol(const char* name) {
     const char* lib_name = (_style & STYLE_LIB_NAMES) ? Profiler::instance()->getLibraryName(name) : NULL;
+    bool report_lib_name_from_symbol = false;
+    if (lib_name == NULL && !endsWith(name, "_[k]", 4)) {
```

Comment:
I figured that this code can be simplified, if instead we modify `CodeCache::add` which adds a final "last resort" symbol as the library name. We could at that point make the "last resort" symbol appear as we want (i.e add `( )`). This will eliminate the extra `findLibraryByName` call. On the other hand, it's a larger change, and I prefer to avoid it.
You can try this change and see in the code if it affects anything else. I'm find with this solution as well.